app/mailers/user_mailer.rb

```diff
@@ -39,6 +39,7 @@ def reservation_status_update(reservation, force = '') # rubocop:disable all
 
     if @status == 'reserved'
       # we only send emails for reserved reservations if it was a request
+      return unless @reservation.flagged?(:request)
```

Comment:
This creates some ugliness in testing because we can no longer send emails for basic reservations, but I think it's more important to have this guard clause